telemetry_api/lib/telemetry_api/traces.ex

```diff
@@ -63,6 +63,7 @@ defmodule TelemetryApi.Traces do
   """
   def register_operator_response(merkle_root, operator_id) do
     with {:ok, operator} <- Operators.get_operator(%{id: operator_id}),
+          :ok <- check_operator_status(operator),
```

Comment:
any reason not to use `Operators.is_registered?` instead of `check_operator_status`here? The intent seems more clear with the `is_registered?` function

Reply:
+1